tests/test_good_sounds.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+
+import numpy as np
+
+from mirdata.datasets import good_sounds
+from tests.test_utils import run_track_tests
+
+
+def test_track():
+    default_trackid = "1"
+    data_home = "tests/resources/mir_datasets/good_sounds"
+    track = good_sounds.Track(default_trackid, data_home=data_home)
+
+    expected_attributes = {
+        'audio_path': 'tests/resources/mir_datasets/good_sounds/good-sounds/sound_files/flute_almudena_reference/akg/0000.wav',
+        'track_id': '1'
+    }
+
+    expected_property_types = {
+        'get_pack_info': dict,
+        'get_ratings_info': list,
+        'get_sound_info': dict,
+        'get_take_info': dict
+    }
+
+    run_track_tests(track, expected_attributes, expected_property_types)
+
+    audio, sr = track.audio
+    assert sr == 22050, "sample rate {} is not 44100".format(sr)
```

Comment:
should it be 22050 or 44100?

Reply:
probably a "freudian slip" from the giantsteps dataset template